controllers/root.js

```diff
@@ -113,11 +126,28 @@ var Root = (function() {
         };
         proxy.sendData(protocol, options, req.body, res);
     };
+    var redir_request_validate = function (req, res, user_info) {
+            var info = {Nick_Name:'',Display_Name:'',Roles:'',Organizations:''};
+            if (user_info) {
+                if (config.tokens_engine === 'keystone') {
+                    info.Nick_Name = user_info.token.user.id;
+            	     info.Display_Name = user_info.token.user.id;
+                    info.Roles = user_info.token.roles;
+                    info.Organizations = user_info.token.project;
+                } else {
+                    info.Nick_Name = user_info.id;
+                    info.Display_Name = user_info.displayName;
+                    info.Roles = user_info.roles;
+                    info.Organizations = user_info.organizations;
+                }
+                res.status(200).send('Access-token OK sending user information: '+JSON.stringify(info));
```

Comment:
The info should be included directly in the response. With no additional text. 

`res.status(200).sendJSON.stringify(info));`